src/App.jsx

```diff
@@ -60,9 +60,20 @@ export function App() {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [listToken]);
 
+	function logOut() {
+		const confirm = window.confirm(
+			`If you're ready to log out, make sure to write down your list name before you click ok! It is ${listToken}`,
```

Comment:
I really like this reminder/confirmation before logout! My only suggestion would be to add quotes, single or double, around the list token. I think it helps distinguish the token from the rest of the text a little easier, kind of similar to how it's a different color on the List component.

Reply:
I agree with @Trevor-Rezac on additional highlighting the list name, may be make it italic or bold if the window.confirm allows it?

Reply:
and I like the notification part, because we care

Reply:
Oh good call! I have added quotes around it. Unfortunately it looks like window.confirm can't be styled on it's own.